tasks/rsync-deploy.yml

```diff
@@ -1,19 +1,24 @@
 ---
+
+# Migration check from symlink deployment
+- stat:
+    path: "{{ ansistrano_deploy_to }}/{{ ansistrano_current_dir }}"
+  register: stat_current_dir
+
+- name: ANSISTRANO | Remove current folder if it's a symlink
+  file:
+    state: absent
+    path: "{{ ansistrano_deploy_to }}/{{ ansistrano_current_dir }}"
+  when: stat_current_dir.stat.islnk is defined and stat_current_dir.stat.islnk
+
 # Perform rsync deployment
 - name: ANSISTRANO | Ensure current folder is a directory
   file:
     state: directory
     path: "{{ ansistrano_deploy_to }}/{{ ansistrano_current_dir }}"
 
 - name: ANSISTRANO | Sync release to new current path
-  synchronize:
-    src: "{{ ansistrano_release_path.stdout }}/"
-    dest: "{{ ansistrano_deploy_to }}/{{ ansistrano_current_dir }}/"
-    set_remote_user: "{{ ansistrano_rsync_set_remote_user }}"
-    recursive: yes
-    delete: yes
-    archive: yes
-  delegate_to: "{{ inventory_hostname }}"
+  command: rsync -a -F --delete-after "{{ ansistrano_release_path.stdout }}/" "{{ ansistrano_deploy_to }}/{{ ansistrano_current_dir }}/"
```

Comment:
I see, I think there is some parameter to avoid this known_hosts file situation but I can´t remember and we can always sort it out later

I need to do some tests with the .rsync-filter as it has been a while since I don´t dig into all the options we have :)
Do you want to give a go to rollback? Or at least review what we would do there?